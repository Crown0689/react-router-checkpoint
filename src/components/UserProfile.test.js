import React from 'react'
import renderer from 'react-test-renderer'
import UserProfile from './UserProfile'


const user = {
    name: "dominik",
    email: "test@mail.de",
    company: "volkswagen",
    phone: "123454"
}

test('UserProfile renders consistently', () => {
    const tree = renderer.create(<UserProfile.WrappedComponent user={user} />).toJSON()
    expect(tree).toMatchSnapshot()
})


