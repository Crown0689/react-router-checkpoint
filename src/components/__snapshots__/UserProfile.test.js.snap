// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile renders consistently 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col col-sm-6 offset-sm-3"
      >
        <h1
          className="text-center"
        >
          User Profile Page
        </h1>
      </div>
    </div>
    <div
      className="row"
      style={
        Object {
          "marginTop": 20,
        }
      }
    >
      <div
        className="col"
      >
        <img
          alt="profile"
          src="http://via.placeholder.com/350x450"
        />
      </div>
      <div
        className="col"
      >
        <h3>
          Name: 
          dominik
        </h3>
        <h3>
          Email: 
          test@mail.de
        </h3>
        <h3>
          Company: 
          volkswagen
        </h3>
        <h3>
          Phone: 
          123454
        </h3>
        <h3>
          Address: 
        </h3>
      </div>
    </div>
  </div>
</div>
`;
